core/src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -527,6 +528,10 @@ public void onFileInit(Path file) {
             if (logger.isTraceEnabled()) {
                 logger.trace("Loading script file : [{}]", file);
             }
+            if (FileSystemUtils.isHidden(file)) {
+                logger.warn("--- Hidden file skipped : [{}]", file);
```

Comment:
I'm not sure if warn is the right level here, maybe debug. Can you remove the leading "--- " and the word "script" to the log message somewhere?